packages/lumx-react/src/components/avatar/Avatar.tsx

```diff
@@ -36,6 +36,7 @@ export interface AvatarProps extends GenericProps {
         ThumbnailProps,
         'image' | 'alt' | 'size' | 'theme' | 'align' | 'fillHeight' | 'variant' | 'aspectRatio'
     >;
+    linkProps?: any;
```

Comment:
Missing JSDoc. Something like `Props to pass to the link wrapping the thumbnail`

Reply:
Also I'd rather have the `linkAs` prop outside the `linkProps` like other components (`ListItem`, `SideNavigationItem`) so that it can be documented here